ggml/CMakeLists.txt

```diff
@@ -166,6 +166,7 @@ option(GGML_SYCL                            "ggml: use SYCL"
 option(GGML_SYCL_F16                        "ggml: use 16 bit floats for sycl calculations"   OFF)
 set   (GGML_SYCL_TARGET "INTEL" CACHE STRING
                                             "ggml: sycl target device")
+set   (GGML_SYCL_ARCH "" CACHE STRING       "ggml: sycl architecture")
```

Comment:
```suggestion
set   (GGML_SYCL_DEVICE_ARCH "" CACHE STRING       "ggml: sycl device architecture")
```

Reply:
Done in https://github.com/ggerganov/llama.cpp/pull/10266/commits/5430726711ae12e17800297c15acf1495043c5ab